### ISSUE TITLE
Support notification strings

### ISSUE THREAD

**Issue by Etheroit** [blake-mealey/mantle#212]
**What type of feature request is this?**
- [x] Mantle doesn't support a Roblox feature I would like to use
- [ ] Mantle CLI improvement
- [ ] Other

**Describe your problem**
It would be awesome if you could create notification strings just like you can manage gamepasses etc.
Please see:
- https://create.roblox.com/docs/production/promotion/invite-prompts
- https://devforum.roblox.com/t/introducing-experience-notifications/2826474

**Describe the solution you'd like**

**Comment by blake-mealey** [comment]
There is already some support for this. Is there anything missing here that you still need? See docs: https://mantledeploy.vercel.app/docs/configuration/reference#target-experience-notifications

**Comment by Etheroit** [comment]
> There is already some support for this. Is there anything missing here that you still need? See docs: https://mantledeploy.vercel.app/docs/configuration/reference#target-experience-notifications

Thanks for a quick response. My fault again. I've been browsing through some docs and remembered the feature was released ~5 days ago, so I gave it a search in docs and nothing appeared.
<img width="438" alt="image" src="https://github.com/blake-mealey/mantle/assets/42714242/b9821264-32bd-49fb-8bca-6ed55a927945">
Should have checked more carefully. I've tested and indeed it works like it's supposed to and is implemented.
